misc/pyinstaller.md

```diff
@@ -0,0 +1,26 @@
+# Building Windows executable (exe)
```

Comment:
this shouldn't be part of this PR

Reply:
Not sure how this happened. Will open a new one